var/spack/repos/builtin/packages/xrootd/package.py

```diff
@@ -94,6 +95,18 @@ class Xrootd(CMakePackage):
         sha256="2655e2d609d80bf9c9ab58557f4f6940408a1af9c686e7aa214ac0348c89c8fa",
         when="@5.5.1",
     )
+    # https://github.com/xrootd/xrootd/pull/1930
+    patch(
+        "https://patch-diff.githubusercontent.com/raw/xrootd/xrootd/pull/1930.patch?full_index=1",
+        sha256="969f8b07edff42449ad76b02f3e57d93b8d6c829be1ba14bccf831c27bc971e1",
+        when="@5.5.3",
+    )
+    # https://github.com/xrootd/xrootd/pull/1929
```

Comment:
Should we be worried about this patch's stability since it hasn't been merged yet? If the branch is rebased before merged, the hash would become invalid.

Reply:
https://github.com/xrootd/xrootd/pull/1930 has been merged, so I will assume you're referring to https://github.com/xrootd/xrootd/pull/1929?

This is a reasonable concern: would you prefer a local patch?

Reply:
Yes, I was referring to 1929. I think we can use a patch-diff link for a specific commit comparison (or a single commit here). I was looking at the discussion which says that this patch will wait until 1933 is merged, but that has been closed and deferred until xrootd-6, and I don't know the timeline for that. So maybe https://github.com/xrootd/xrootd/pull/1929/commits/294ea1c5420c29113fa733615dbe04c706e5e418.patch?full_index=1 ? It is probably still not stable to rebases but at least stable against default branch merges.

Reply:
@wdconinc It appears I had more changes in my local fork than I have included here, and this PR relies on them. I will close, consolidate and resubmit. Sorry for the waste of your time.

Reply:
I think the patch by link is probably still preferred over duplicating the fix. Gah, there are no good choices here, are there?

Reply:
Not really. I went with the `filter_file()` in the end on the grounds that it's analogous to the `filter_file()` removing `-std=c++0x` for an earlier version of XRootD. I can revert if you'd prefer.